rules/manager.go

```diff
@@ -212,8 +212,12 @@ func (g *Group) copyState(from *Group) {
 			if !ok {
 				continue
 			}
-			if far.Name() == ar.Name() {
-				ar.active = far.active
+			// TODO(fabxc): forbid same alert definitions that are not unique by
```

Comment:
We need some sort of unique-ness. Then we might have to add everything else in there too (expression, FOR, annotations). Otherwise the internal state is ambiguous.